open-metadata-distribution/omag-server-platform/docs/http-client-collections/README.md

```diff
@@ -0,0 +1,54 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# Sample Jetbrains HTTP Scripts
+
+The Jetbrains HTTP Client is a useful utility for testing REST APIs that is both integrated into the Jetbrains tooling 
+and freely available to be run standalone. Further information can be found at 
+[Jebrains HTTP Client](https://www.jetbrains.com/help/idea/http-client-in-product-code-editor.html). Information about the freely downloadable client can be found at 
```

Comment:
Do we know what he license it for this ? 
I suspect the the IJ plugin is Ultimate license but the command line version is CE, the closest I found was following the third party software from link https://www.jetbrains.com/ijhttp/download/#section=zip-archive. I was not sure if we were happy with all these licenses. I suggest we be more explicit in the documentation to detail these 2 versions of the http client and their respective licenses.

Reply:
I am trying to understand the motivation behind this series of comments.

The HTTP Client is not a replacement for Postman - it is an alternative - and we are not requiring anyone to use them these new scripts in order to use Egeria.   (Similar comment for Postman.)    This is a different issue from, say our use of Spring, which is built into the platform.

Postman has moved to a cloud service and there are concerns in the community that secrets can be leaked through the use of the postman service.  We agreed on the community call that it would be useful to offer an alternative in our samples that runs locally in IntelliJ, PyCharm etc for organizations with concerns about Postman.  Dan is just implementing the agreement.

Both Postman and these new Http Client scripts are examples, and completely optional.  In the same way we show examples of using Postman, Docker, Rancher etc, it should be enough to link to the home page of the technology's website to allow people to make their own decision on whether the licenses are ok for them.

Licenses (and the pages where they are documented) change over time.  It is possible that we could be providing misleading information on licensing if we try to intepret the licenses for other people.

Is there something more fundamental that I am missing?

Reply:
@mandy-chessell the link that originally was there pointed to IJ Ultimate - so I assumed that this was not available freely - which is what the text indicated. Dan has updated it to include the non Ultimate way of getting this - so it is now not confusing.